code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -21,10 +21,9 @@
 	anchored = TRUE
 
 /obj/structure/closet/emcloset/Initialize(mapload)
-	. = ..()
-
 	if (prob(1))
 		return INITIALIZE_HINT_QDEL
+	return ..()
```

Comment:
This is a bad fix, `..()` should always come first in Initialize

I don't understand your description of the hard del here, either. PopulateContents() *isn't* called in Initialize. That's the whole point of #69587.

The issue is almost certainly coming from the closets test, which does not check if the locker has been qdeleted before populating contents. Add the check there and this bug is fixed.